config/locales/en/coronavirus_landing_page.yml

```diff
@@ -0,0 +1,21 @@
+en:
+  coronavirus_landing_page:
+    video_section_heading: ""
```

Comment:
Should this have some text in it? Currently it's rendering as an empty H2...

<img width="765" alt="Screenshot 2020-09-09 at 13 08 55" src="https://user-images.githubusercontent.com/861310/92596406-acbbd680-f29d-11ea-8eb9-b22082bc35f5.png">

Reply:
Ah, sorry, just saw your comment that you're already aware of this 👍

Reply:
This is the section heading we're still pending – will add the content as soon as we receive it (before merging)